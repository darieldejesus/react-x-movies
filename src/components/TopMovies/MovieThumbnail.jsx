import React from 'react';
import PropTypes from 'prop-types';
import Styled from 'styled-components';

const Container = Styled.div`
  display: block;
  width: 154px;
`;

const Image = Styled.img`
  background: red;
  height: 231px;
  width: 154px;
`;

const MovieName = Styled.a`
  color #63676f;
  font-size: 1rem;
  font-weight: 300;
  margin: 10px 0 0;
  text-decoration: none;
  text-transform: uppercase;
`;

const MovieReleaseDate = Styled.h4`
  color: #ed0005;
  font-size: 0.8rem;
  font-weight: 600;
  margin: 0;
`;

const MovieThumbnail = ({ id, title, poster_path, release_date }) => (
  <Container>
    <Image src={`https://image.tmdb.org/t/p/w154${poster_path}`} />
    <MovieName href={`/movie/${id}`}>{ title }</MovieName>
    <MovieReleaseDate>({ release_date })</MovieReleaseDate>
  </Container>
);

MovieThumbnail.propTypes = {
  id: PropTypes.number.isRequired,
  poster_path: PropTypes.string.isRequired,
  release_date: PropTypes.string.isRequired,
  title: PropTypes.string.isRequired,
};

export default MovieThumbnail;
