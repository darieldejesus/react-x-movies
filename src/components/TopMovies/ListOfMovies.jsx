import React from 'react';
import PropTypes from 'prop-types';
import Styled from 'styled-components';

import MovieThumbnail from './MovieThumbnail';

const Movies = Styled.div`
  display: flex;
  justify-content: space-between;
`;

const ListOfMovies = ({ entries }) => (
  <Movies>
    {entries.slice(0, 6).map(entry => (
      <MovieThumbnail key={entry.title} {...entry} />
    ))}
  </Movies>
);

ListOfMovies.propTypes = {
  entries: PropTypes.arrayOf(PropTypes.shape({
    poster_path: PropTypes.string.isRequired,
    release_date: PropTypes.string.isRequired,
    title: PropTypes.string.isRequired,
  })).isRequired,
};

export default ListOfMovies;
