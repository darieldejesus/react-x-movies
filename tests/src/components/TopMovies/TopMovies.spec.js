import React from 'react';
import TopMovies from '../../../../src/components/TopMovies/TopMovies';
import fixture from '../../../fixture/movies.json';

describe('TopMovies', function () {
  beforeEach(() => {
    this.wrapper = shallow(<TopMovies entries={fixture} />);
  });

  test('should renders ListOfMovies', () => {
    expect(toJson(this.wrapper)).toMatchSnapshot();
  });
});

