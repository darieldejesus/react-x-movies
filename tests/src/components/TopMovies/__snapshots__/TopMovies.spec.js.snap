// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopMovies should renders ListOfMovies 1`] = `
<styled.section>
  <styled.h1
    key="title"
  >
    Top 6 Movies
  </styled.h1>
  ,
  <ListOfMovies
    entries={
      Array [
        Object {
          "adult": false,
          "backdrop_path": "/tcheoA2nPATCm2vvXw2hVQoaEFD.jpg",
          "genre_ids": Array [
            27,
            53,
          ],
          "id": 346364,
          "original_language": "en",
          "original_title": "It",
          "overview": "In a small town in Maine, seven children known as The Losers Club come face to face with life problems, bullies and a monster that takes the shape of a clown called Pennywise.",
          "popularity": 729.612854,
          "poster_path": "/9E2y5Q7WlCVNEhP5GiVTjhEhx1o.jpg",
          "release_date": "2017-09-05",
          "title": "It",
          "video": false,
          "vote_average": 7.3,
          "vote_count": 3559,
        },
        Object {
          "adult": false,
          "backdrop_path": "/wBzMnQ01R9w58W6ucltdYfOyP4j.jpg",
          "genre_ids": Array [
            28,
            12,
            35,
            14,
            878,
          ],
          "id": 284053,
          "original_language": "en",
          "original_title": "Thor: Ragnarok",
          "overview": "Thor is imprisoned on the other side of the universe and finds himself in a race against time to get back to Asgard to stop Ragnarok, the destruction of his homeworld and the end of Asgardian civilization, at the hands of an all-powerful new threat, the ruthless Hela.",
          "popularity": 627.309751,
          "poster_path": "/oSLd5GYGsiGgzDPKTwQh7wamO8t.jpg",
          "release_date": "2017-10-25",
          "title": "Thor: Ragnarok",
          "video": false,
          "vote_average": 7.7,
          "vote_count": 1297,
        },
        Object {
          "adult": false,
          "backdrop_path": "/eGx5OfOdvM0gkHdmkLe3hcJuEIT.jpg",
          "genre_ids": Array [
            27,
            9648,
            53,
          ],
          "id": 440021,
          "original_language": "en",
          "original_title": "Happy Death Day",
          "overview": "A college student relives the day of her murder over and over again as she tries to discover her killer's identity.",
          "popularity": 560.461034,
          "poster_path": "/cTaEIUYTt52ooq9quVbAQ7NpGwo.jpg",
          "release_date": "2017-10-12",
          "title": "Happy Death Day",
          "video": false,
          "vote_average": 6.5,
          "vote_count": 243,
        },
        Object {
          "adult": false,
          "backdrop_path": "/uX7LXnsC7bZJZjn048UCOwkPXWJ.jpg",
          "genre_ids": Array [
            10751,
            16,
            12,
            35,
          ],
          "id": 211672,
          "original_language": "en",
          "original_title": "Minions",
          "overview": "Minions Stuart, Kevin and Bob are recruited by Scarlet Overkill, a super-villain who, alongside her inventor husband Herb, hatches a plot to take over the world.",
          "popularity": 444.785132,
          "poster_path": "/q0R4crx2SehcEEQEkYObktdeFy.jpg",
          "release_date": "2015-06-17",
          "title": "Minions",
          "video": false,
          "vote_average": 6.4,
          "vote_count": 4951,
        },
        Object {
          "adult": false,
          "backdrop_path": "/2SEgJ0mHJ7TSdVDbkGU061tR33K.jpg",
          "genre_ids": Array [
            18,
            53,
            28,
            878,
          ],
          "id": 347882,
          "original_language": "en",
          "original_title": "Sleight",
          "overview": "A young street magician is left to take care of his little sister after his mother's passing and turns to drug dealing in the Los Angeles party scene to keep a roof over their heads. When he gets into trouble with his supplier, his sister is kidnapped and he is forced to rely on both his sleight of hand and brilliant mind to save her.",
          "popularity": 443.860101,
          "poster_path": "/wridRvGxDqGldhzAIh3IcZhHT5F.jpg",
          "release_date": "2017-04-28",
          "title": "Sleight",
          "video": false,
          "vote_average": 5.5,
          "vote_count": 94,
        },
        Object {
          "adult": false,
          "backdrop_path": "/ulMscezy9YX0bhknvJbZoUgQxO5.jpg",
          "genre_ids": Array [
            18,
            878,
            10752,
          ],
          "id": 281338,
          "original_language": "en",
          "original_title": "War for the Planet of the Apes",
          "overview": "Caesar and his apes are forced into a deadly conflict with an army of humans led by a ruthless Colonel. After the apes suffer unimaginable losses, Caesar wrestles with his darker instincts and begins his own mythic quest to avenge his kind. As the journey finally brings them face to face, Caesar and the Colonel are pitted against each other in an epic battle that will determine the fate of both their species and the future of the planet.",
          "popularity": 404.116934,
          "poster_path": "/3vYhLLxrTtZLysXtIWktmd57Snv.jpg",
          "release_date": "2017-07-11",
          "title": "War for the Planet of the Apes",
          "video": false,
          "vote_average": 6.8,
          "vote_count": 2231,
        },
        Object {
          "adult": false,
          "backdrop_path": "/vc8bCGjdVp0UbMNLzHnHSLRbBWQ.jpg",
          "genre_ids": Array [
            28,
            12,
            35,
            878,
          ],
          "id": 315635,
          "original_language": "en",
          "original_title": "Spider-Man: Homecoming",
          "overview": "Following the events of Captain America: Civil War, Peter Parker, with the help of his mentor Tony Stark, tries to balance his life as an ordinary high school student in Queens, New York City, with fighting crime as his superhero alter ego Spider-Man as a new threat, the Vulture, emerges.",
          "popularity": 400.437774,
          "poster_path": "/kY2c7wKgOfQjvbqe7yVzLTYkxJO.jpg",
          "release_date": "2017-07-05",
          "title": "Spider-Man: Homecoming",
          "video": false,
          "vote_average": 7.3,
          "vote_count": 4436,
        },
        Object {
          "adult": false,
          "backdrop_path": "/mVr0UiqyltcfqxbAUcLl9zWL8ah.jpg",
          "genre_ids": Array [
            28,
            9648,
            878,
            53,
          ],
          "id": 335984,
          "original_language": "en",
          "original_title": "Blade Runner 2049",
          "overview": "Thirty years after the events of the first film, a new blade runner, LAPD Officer K, unearths a long-buried secret that has the potential to plunge what's left of society into chaos. K's discovery leads him on a quest to find Rick Deckard, a former LAPD blade runner who has been missing for 30 years.",
          "popularity": 319.63386,
          "poster_path": "/aMpyrCizvSdc0UIMblJ1srVgAEF.jpg",
          "release_date": "2017-10-04",
          "title": "Blade Runner 2049",
          "video": false,
          "vote_average": 7.4,
          "vote_count": 1625,
        },
        Object {
          "adult": false,
          "backdrop_path": "/ytKpFaLMpFWnuSXStz1GHrtTt6R.jpg",
          "genre_ids": Array [
            27,
            53,
          ],
          "id": 298250,
          "original_language": "en",
          "original_title": "Jigsaw",
          "overview": "Dead bodies begin to turn up all over the city, each meeting their demise in a variety of grisly ways. All investigations begin to point the finger at deceased killer John Kramer.",
          "popularity": 286.213904,
          "poster_path": "/zUbUtxiTdEgWnkXY945gtYYqBZ1.jpg",
          "release_date": "2017-10-20",
          "title": "Jigsaw",
          "video": false,
          "vote_average": 5.9,
          "vote_count": 187,
        },
        Object {
          "adult": false,
          "backdrop_path": "/lhTtnsPmEYUJB7nOaTKJzYoxJ7X.jpg",
          "genre_ids": Array [
            28,
            878,
            53,
          ],
          "id": 274855,
          "original_language": "en",
          "original_title": "Geostorm",
          "overview": "Gerard Butler playing a stubborn but charming satellite designer who, when the worldâ€™s climate-controlling satellites malfunction, has to work together with his estranged brother to save the world from a man-made storm of epic proportions. A trip into space follows, while on Earth a plot to assassinate the president begins to unfold.",
          "popularity": 268.44468,
          "poster_path": "/nrsx0jEaBgXq4PWo7SooSnYJTv.jpg",
          "release_date": "2017-10-13",
          "title": "Geostorm",
          "video": false,
          "vote_average": 5.8,
          "vote_count": 228,
        },
        Object {
          "adult": false,
          "backdrop_path": "/6aUWe0GSl69wMTSWWexsorMIvwU.jpg",
          "genre_ids": Array [
            10751,
            14,
            10749,
          ],
          "id": 321612,
          "original_language": "en",
          "original_title": "Beauty and the Beast",
          "overview": "A live-action adaptation of Disney's version of the classic 'Beauty and the Beast' tale of a cursed prince and a beautiful young woman who helps him break the spell.",
          "popularity": 266.756309,
          "poster_path": "/tWqifoYuwLETmmasnGHO7xBjEtt.jpg",
          "release_date": "2017-03-16",
          "title": "Beauty and the Beast",
          "video": false,
          "vote_average": 6.8,
          "vote_count": 5860,
        },
        Object {
          "adult": false,
          "backdrop_path": "/2BXd0t9JdVqCp9sKf6kzMkr7QjB.jpg",
          "genre_ids": Array [
            12,
            10751,
            16,
            28,
            35,
          ],
          "id": 177572,
          "original_language": "en",
          "original_title": "Big Hero 6",
          "overview": "The special bond that develops between plus-sized inflatable robot Baymax, and prodigy Hiro Hamada, who team up with a group of friends to form a band of high-tech heroes.",
          "popularity": 243.34689,
          "poster_path": "/9gLu47Zw5ertuFTZaxXOvNfy78T.jpg",
          "release_date": "2014-10-24",
          "title": "Big Hero 6",
          "video": false,
          "vote_average": 7.8,
          "vote_count": 6494,
        },
        Object {
          "adult": false,
          "backdrop_path": "/tAg6KUBANIVbYUpTHy5oKibhhw3.jpg",
          "genre_ids": Array [
            80,
            18,
            27,
            9648,
            53,
          ],
          "id": 372343,
          "original_language": "en",
          "original_title": "The Snowman",
          "overview": "Detective Harry Hole investigates the disappearance of a woman whose pink scarf is found wrapped around an ominous looking snowman.",
          "popularity": 242.872027,
          "poster_path": "/bQHgpTVsAWjNQWS0frsl7DlzLX1.jpg",
          "release_date": "2017-10-12",
          "title": "The Snowman",
          "video": false,
          "vote_average": 5,
          "vote_count": 162,
        },
        Object {
          "adult": false,
          "backdrop_path": "/6iUNJZymJBMXXriQyFZfLAKnjO6.jpg",
          "genre_ids": Array [
            28,
            12,
            14,
          ],
          "id": 297762,
          "original_language": "en",
          "original_title": "Wonder Woman",
          "overview": "An Amazon princess comes to the world of Man to become the greatest of the female superheroes.",
          "popularity": 234.426933,
          "poster_path": "/imekS7f1OuHyUP2LAiTEM0zBzUz.jpg",
          "release_date": "2017-05-30",
          "title": "Wonder Woman",
          "video": false,
          "vote_average": 7.2,
          "vote_count": 5597,
        },
        Object {
          "adult": false,
          "backdrop_path": "/go1a6YwrVTMrNNIfaRpBRp5pIzV.jpg",
          "genre_ids": Array [
            12,
            18,
            53,
          ],
          "id": 390062,
          "original_language": "en",
          "original_title": "Jungle",
          "overview": "A mysterious guide escorts an enthusiastic adventurer and his friend into the Amazon jungle, but their journey turns into a terrifying ordeal as the darkest elements of human nature and the deadliest threats of the wild force them to fight for survival.",
          "popularity": 217.217984,
          "poster_path": "/tDgxknTVwrScxpCYyGUjXSn5NRk.jpg",
          "release_date": "2017-10-20",
          "title": "Jungle",
          "video": false,
          "vote_average": 5.3,
          "vote_count": 111,
        },
        Object {
          "adult": false,
          "backdrop_path": "/lj3wkYkJGkfmDfXkKrVarSJpFD4.jpg",
          "genre_ids": Array [
            53,
            9648,
            28,
          ],
          "id": 341013,
          "original_language": "en",
          "original_title": "Atomic Blonde",
          "overview": "An undercover MI6 agent is sent to Berlin during the Cold War to investigate the murder of a fellow agent and recover a missing list of double agents.",
          "popularity": 210.380053,
          "poster_path": "/kV9R5h0Yct1kR8Hf8sJ1nX0Vz4x.jpg",
          "release_date": "2017-07-26",
          "title": "Atomic Blonde",
          "video": false,
          "vote_average": 6.2,
          "vote_count": 1011,
        },
        Object {
          "adult": false,
          "backdrop_path": "/umC04Cozevu8nn3JTDJ1pc7PVTn.jpg",
          "genre_ids": Array [
            28,
            53,
          ],
          "id": 245891,
          "original_language": "en",
          "original_title": "John Wick",
          "overview": "Ex-hitman John Wick comes out of retirement to track down the gangsters that took everything from him.",
          "popularity": 194.137234,
          "poster_path": "/5vHssUeVe25bMrof1HyaPyWgaP.jpg",
          "release_date": "2014-10-22",
          "title": "John Wick",
          "video": false,
          "vote_average": 7,
          "vote_count": 5670,
        },
        Object {
          "adult": false,
          "backdrop_path": "/bt6DhdALyhf90gReozoQ0y3R3vZ.jpg",
          "genre_ids": Array [
            9648,
            53,
            18,
          ],
          "id": 210577,
          "original_language": "en",
          "original_title": "Gone Girl",
          "overview": "With his wife's disappearance having become the focus of an intense media circus, a man sees the spotlight turned on him when it's suspected that he may not be innocent.",
          "popularity": 169.391851,
          "poster_path": "/gdiLTof3rbPDAmPaCf4g6op46bj.jpg",
          "release_date": "2014-10-01",
          "title": "Gone Girl",
          "video": false,
          "vote_average": 7.9,
          "vote_count": 6318,
        },
        Object {
          "adult": false,
          "backdrop_path": "/4PwyB0ErucIANzW24Kori71J6gU.jpg",
          "genre_ids": Array [
            878,
            12,
            53,
          ],
          "id": 131631,
          "original_language": "en",
          "original_title": "The Hunger Games: Mockingjay - Part 1",
          "overview": "Katniss Everdeen reluctantly becomes the symbol of a mass rebellion against the autocratic Capitol.",
          "popularity": 157.549327,
          "poster_path": "/gj282Pniaa78ZJfbaixyLXnXEDI.jpg",
          "release_date": "2014-11-18",
          "title": "The Hunger Games: Mockingjay - Part 1",
          "video": false,
          "vote_average": 6.6,
          "vote_count": 6013,
        },
        Object {
          "adult": false,
          "backdrop_path": "/9rZg1J6vMQoDVSgRyWcpJa8IAGy.jpg",
          "genre_ids": Array [
            53,
            80,
          ],
          "id": 680,
          "original_language": "en",
          "original_title": "Pulp Fiction",
          "overview": "A burger-loving hit man, his philosophical partner, a drug-addled gangster's moll and a washed-up boxer converge in this sprawling, comedic crime caper. Their adventures unfurl in three stories that ingeniously trip back and forth in time.",
          "popularity": 154.702446,
          "poster_path": "/dM2w364MScsjFf8pfMbaWUcWrR.jpg",
          "release_date": "1994-09-10",
          "title": "Pulp Fiction",
          "video": false,
          "vote_average": 8.3,
          "vote_count": 9118,
        },
        Object {
          "adult": false,
          "backdrop_path": "/78kN1a1AHxAPEh7hQhNjDQhLZ9W.jpg",
          "genre_ids": Array [
            27,
            35,
          ],
          "id": 459202,
          "original_language": "en",
          "original_title": "Boo 2! A Madea Halloween",
          "overview": "Madea and the gang encounter monsters, goblins and boogeymen at a haunted campground.",
          "popularity": 131.573297,
          "poster_path": "/5nvP4etJ8ecQv8qZM08dK4BAzdK.jpg",
          "release_date": "2017-10-20",
          "title": "Boo 2! A Madea Halloween",
          "video": false,
          "vote_average": 5.4,
          "vote_count": 85,
        },
        Object {
          "adult": false,
          "backdrop_path": "/oakHIN1th8DefPrMu2Eo52qHhnS.jpg",
          "genre_ids": Array [
            18,
            27,
            9648,
            53,
          ],
          "id": 399057,
          "original_language": "en",
          "original_title": "The Killing of a Sacred Deer",
          "overview": "A teenager's attempts to bring a brilliant surgeon into his dysfunctional family take an unexpected turn.",
          "popularity": 129.25268,
          "poster_path": "/e4DGlsc9g0h5AyoyvvAuIRnofN7.jpg",
          "release_date": "2017-10-20",
          "title": "The Killing of a Sacred Deer",
          "video": false,
          "vote_average": 5.8,
          "vote_count": 48,
        },
        Object {
          "adult": false,
          "backdrop_path": "/3FweBee0xZoY77uO1bhUOlQorNH.jpg",
          "genre_ids": Array [
            28,
            12,
            14,
          ],
          "id": 76338,
          "original_language": "en",
          "original_title": "Thor: The Dark World",
          "overview": "Thor fights to restore order across the cosmosâ€¦ but an ancient race led by the vengeful Malekith returns to plunge the universe back into darkness. Faced with an enemy that even Odin and Asgard cannot withstand, Thor must embark on his most perilous and personal journey yet, one that will reunite him with Jane Foster and force him to sacrifice everything to save us all.",
          "popularity": 124.775567,
          "poster_path": "/bnX5PqAdQZRXSw3aX3DutDcdso5.jpg",
          "release_date": "2013-10-29",
          "title": "Thor: The Dark World",
          "video": false,
          "vote_average": 6.7,
          "vote_count": 5144,
        },
        Object {
          "adult": false,
          "backdrop_path": "/p7mUT8Usu1kNlfd1ugk6FwKejlo.jpg",
          "genre_ids": Array [
            10749,
            12,
            18,
          ],
          "id": 290512,
          "original_language": "en",
          "original_title": "The Mountain Between Us",
          "overview": "Stranded after a tragic plane crash, two strangers must forge a connection to survive the extreme elements of a remote snow covered mountain. When they realize help is not coming, they embark on a perilous journey across the wilderness.",
          "popularity": 122.804156,
          "poster_path": "/3XNfYTW4XGscI81nXMSWGsQ8cpu.jpg",
          "release_date": "2017-10-05",
          "title": "The Mountain Between Us",
          "video": false,
          "vote_average": 6.4,
          "vote_count": 72,
        },
        Object {
          "adult": false,
          "backdrop_path": "/2Qwb7EayKsD2cuB3I2u2k2VU3Lb.jpg",
          "genre_ids": Array [
            12,
            28,
            14,
            35,
          ],
          "id": 166426,
          "original_language": "en",
          "original_title": "Pirates of the Caribbean: Dead Men Tell No Tales",
          "overview": "Thrust into an all-new adventure, a down-on-his-luck Capt. Jack Sparrow feels the winds of ill-fortune blowing even more strongly when deadly ghost sailors led by his old nemesis, the evil Capt. Salazar, escape from the Devil's Triangle. Jack's only hope of survival lies in seeking out the legendary Trident of Poseidon, but to find it, he must forge an uneasy alliance with a brilliant and beautiful astronomer and a headstrong young man in the British navy.",
          "popularity": 120.067194,
          "poster_path": "/xbpSDU3p7YUGlu9Mr6Egg2Vweto.jpg",
          "release_date": "2017-05-23",
          "title": "Pirates of the Caribbean: Dead Men Tell No Tales",
          "video": false,
          "vote_average": 6.6,
          "vote_count": 3146,
        },
        Object {
          "adult": false,
          "backdrop_path": "/egHeYqgrutiRAzGGWTKp3zqUw0j.jpg",
          "genre_ids": Array [
            18,
            28,
            80,
            9648,
          ],
          "id": 395458,
          "original_language": "en",
          "original_title": "Suburbicon",
          "overview": "A crime mystery set in the quiet family town of Suburbicon during the 1950s, where the best and worst of humanity is hilariously reflected through the deeds of seemingly ordinary people. When a home invasion turns deadly, a picture-perfect family turns to blackmail, revenge, and murder.",
          "popularity": 119.910582,
          "poster_path": "/a3IHgSwO5jWPLcGjKqbQ7pxVGkq.jpg",
          "release_date": "2017-10-27",
          "title": "Suburbicon",
          "video": false,
          "vote_average": 4.8,
          "vote_count": 27,
        },
        Object {
          "adult": false,
          "backdrop_path": "/rND0GSv8X2Zzs7VhyFAs9BCEmYE.jpg",
          "genre_ids": Array [
            10751,
            35,
            16,
            12,
          ],
          "id": 260514,
          "original_language": "en",
          "original_title": "Cars 3",
          "overview": "Blindsided by a new generation of blazing-fast racers, the legendary Lightning McQueen is suddenly pushed out of the sport he loves. To get back in the game, he will need the help of an eager young race technician with her own plan to win, inspiration from the late Fabulous Hudson Hornet, and a few unexpected turns. Proving that #95 isn't through yet will test the heart of a champion on Piston Cup Racingâ€™s biggest stage!",
          "popularity": 115.174703,
          "poster_path": "/fyy1nDC8wm553FCiBDojkJmKLCs.jpg",
          "release_date": "2017-06-15",
          "title": "Cars 3",
          "video": false,
          "vote_average": 6.7,
          "vote_count": 894,
        },
        Object {
          "adult": false,
          "backdrop_path": "/LvmmDZxkTDqp0DX7mUo621ahdX.jpg",
          "genre_ids": Array [
            12,
            14,
            28,
          ],
          "id": 10195,
          "original_language": "en",
          "original_title": "Thor",
          "overview": "Against his father Odin's will, The Mighty Thor - a powerful but arrogant warrior god - recklessly reignites an ancient war. Thor is cast down to Earth and forced to live among humans as punishment. Once here, Thor learns what it takes to be a true hero when the most dangerous villain of his world sends the darkest forces of Asgard to invade Earth.",
          "popularity": 101.349373,
          "poster_path": "/bIuOWTtyFPjsFDevqvF3QrD1aun.jpg",
          "release_date": "2011-04-21",
          "title": "Thor",
          "video": false,
          "vote_average": 6.6,
          "vote_count": 6881,
        },
        Object {
          "adult": false,
          "backdrop_path": "/eUSNKg5tlocYAQs5nHlDxyAefuB.jpg",
          "genre_ids": Array [
            18,
          ],
          "id": 383709,
          "original_language": "en",
          "original_title": "Wonderstruck",
          "overview": "The story of a young boy in the Midwest is told simultaneously with a tale about a young girl in New York from fifty years ago as they both seek the same mysterious connection.",
          "popularity": 98.657961,
          "poster_path": "/yjDAt9K3FKU1itABgIXl6YvLsVE.jpg",
          "release_date": "2017-10-20",
          "title": "Wonderstruck",
          "video": false,
          "vote_average": 5.1,
          "vote_count": 15,
        },
        Object {
          "adult": false,
          "backdrop_path": "/8wI0M7HDHhDsoxMCkcAVUx7O6DX.jpg",
          "genre_ids": Array [
            18,
          ],
          "id": 395991,
          "original_language": "en",
          "original_title": "Only the Brave",
          "overview": "Members of the Granite Mountain Hotshots battle deadly wildfires to save an Arizona town.",
          "popularity": 96.473688,
          "poster_path": "/lC7WdUNLOJI3sllaDGNdFy2GT8g.jpg",
          "release_date": "2017-10-19",
          "title": "Only the Brave",
          "video": false,
          "vote_average": 4.8,
          "vote_count": 32,
        },
        Object {
          "adult": false,
          "backdrop_path": "/Ytv7P13rbwQ3mLpCAY8lBTqI5s.jpg",
          "genre_ids": Array [
            28,
            878,
            53,
            12,
          ],
          "id": 335988,
          "original_language": "en",
          "original_title": "Transformers: The Last Knight",
          "overview": "Autobots and Decepticons are at war, with humans on the sidelines. Optimus Prime is gone. The key to saving our future lies buried in the secrets of the past, in the hidden history of Transformers on Earth.",
          "popularity": 93.088139,
          "poster_path": "/s5HQf2Gb3lIO2cRcFwNL9sn1o1o.jpg",
          "release_date": "2017-06-16",
          "title": "Transformers: The Last Knight",
          "video": false,
          "vote_average": 6.2,
          "vote_count": 1594,
        },
        Object {
          "adult": false,
          "backdrop_path": "/p57KuXI3GbdBEyQ26ImwMRNGyIc.jpg",
          "genre_ids": Array [
            28,
            80,
            53,
          ],
          "id": 398175,
          "original_language": "en",
          "original_title": "Brawl in Cell Block 99",
          "overview": "After working as a drug courier and getting into a brutal shootout with police, a former boxer finds himself at the mercy of his enemies as they force him to instigate violent acts that turn the prison he resides in into a battleground.",
          "popularity": 92.149114,
          "poster_path": "/bfB1J6jsjdGWKjXxKQ5hNd1OyAs.jpg",
          "release_date": "2017-10-06",
          "title": "Brawl in Cell Block 99",
          "video": false,
          "vote_average": 6.6,
          "vote_count": 43,
        },
        Object {
          "adult": false,
          "backdrop_path": "/pVVobDO8cezhVPvwD6EBUN0g3mt.jpg",
          "genre_ids": Array [
            28,
            37,
            878,
            14,
            27,
          ],
          "id": 353491,
          "original_language": "en",
          "original_title": "The Dark Tower",
          "overview": "The last Gunslinger, Roland Deschain, has been locked in an eternal battle with Walter Oâ€™Dim, also known as the Man in Black, determined to prevent him from toppling the Dark Tower, which holds the universe together. With the fate of the worlds at stake, good and evil will collide in the ultimate battle as only Roland can defend the Tower from the Man in Black.",
          "popularity": 90.689856,
          "poster_path": "/i9GUSgddIqrroubiLsvvMRYyRy0.jpg",
          "release_date": "2017-08-03",
          "title": "The Dark Tower",
          "video": false,
          "vote_average": 5.6,
          "vote_count": 1003,
        },
        Object {
          "adult": false,
          "backdrop_path": "/goCvLSUFz0p7k8R10Hv4CVh3EQv.jpg",
          "genre_ids": Array [
            28,
            80,
          ],
          "id": 339403,
          "original_language": "en",
          "original_title": "Baby Driver",
          "overview": "After being coerced into working for a crime boss, a young getaway driver finds himself taking part in a heist doomed to fail.",
          "popularity": 90.689783,
          "poster_path": "/dN9LbVNNZFITwfaRjl4tmwGWkRg.jpg",
          "release_date": "2017-06-28",
          "title": "Baby Driver",
          "video": false,
          "vote_average": 7.2,
          "vote_count": 2615,
        },
        Object {
          "adult": false,
          "backdrop_path": "/n6tHeze754dRHelAF1Vvibjk9t1.jpg",
          "genre_ids": Array [
            99,
            36,
          ],
          "id": 138129,
          "original_language": "en",
          "original_title": "Chiara e Francesco",
          "overview": "Italy, XIII century. The story of friendship between two young people who dedicated their lives to God and others: St. Francis and St. Clare of Assisi. Children of the bourgeoisie and the nobility, respectively, gave up their lives accommodated by a life of sacrifice, humility and proselytizing ...",
          "popularity": 89.494222,
          "poster_path": "/jjRsfwMb1cojHjaY3vZsdMy9fIa.jpg",
          "release_date": "2007-01-01",
          "title": "Chiara e Francesco",
          "video": false,
          "vote_average": 2.5,
          "vote_count": 1,
        },
        Object {
          "adult": false,
          "backdrop_path": "/lMDyuHzBhx3zNAv48vYzmgcJCCD.jpg",
          "genre_ids": Array [
            18,
            35,
          ],
          "id": 419680,
          "original_language": "en",
          "original_title": "Daddy's Home 2",
          "overview": "Brad and Dusty must deal with their intrusive fathers during the holidays.",
          "popularity": 87.524085,
          "poster_path": "/rF2IoKL0IFmumEXQFUuB8LajTYP.jpg",
          "release_date": "2017-11-09",
          "title": "Daddy's Home 2",
          "video": false,
          "vote_average": 5.4,
          "vote_count": 9,
        },
        Object {
          "adult": false,
          "backdrop_path": "/o5T8rZxoWSBMYwjsUFUqTt6uMQB.jpg",
          "genre_ids": Array [
            28,
            12,
            14,
            878,
          ],
          "id": 141052,
          "original_language": "en",
          "original_title": "Justice League",
          "overview": "Fueled by his restored faith in humanity and inspired by Superman's selfless act, Bruce Wayne and Diana Prince assemble a team of metahumans consisting of Barry Allen, Arthur Curry, and Victor Stone to face the catastrophic threat of Steppenwolf and the Parademons who are on the hunt for three Mother Boxes on Earth.",
          "popularity": 84.40271,
          "poster_path": "/9rtrRGeRnL0JKtu9IMBWsmlmmZz.jpg",
          "release_date": "2017-11-15",
          "title": "Justice League",
          "video": false,
          "vote_average": 7.2,
          "vote_count": 6,
        },
        Object {
          "adult": false,
          "backdrop_path": "/5D8DfGzbNzjbvs3mvw4MTCSPwKw.jpg",
          "genre_ids": Array [
            18,
          ],
          "id": 300687,
          "original_language": "en",
          "original_title": "Same Kind of Different as Me",
          "overview": "International art dealer Ron Hall must befriend a dangerous homeless man in order to save his struggling marriage to his wife, a woman whose dreams will lead all three of them on the journey of their lives.",
          "popularity": 83.148844,
          "poster_path": "/cCTwHtXBVL96ImKDwm5T8pPNO41.jpg",
          "release_date": "2017-10-20",
          "title": "Same Kind of Different as Me",
          "video": false,
          "vote_average": 5.5,
          "vote_count": 5,
        },
        Object {
          "adult": false,
          "backdrop_path": "/zms098KnZ30YfV9K0FaeiCUGuQ6.jpg",
          "genre_ids": Array [
            53,
            18,
          ],
          "id": 413362,
          "original_language": "en",
          "original_title": "Roman J. Israel, Esq.",
          "overview": "Hard-nosed liberal lawyer Roman J. Israel has been fighting the good fight forever while others take the credit. When his partner, the firmâ€™s frontman, has a heart attack, Israel suddenly takes on that role. He soon discovers some unsettling truths about the firm--truths that conflict with his values of helping the poor and dispossessed--and finds himself in an existential crisis that leads to extreme actions.",
          "popularity": 78.892306,
          "poster_path": "/8e5IGlLrVjwrlDcRtzSQkXhJFWl.jpg",
          "release_date": "2017-11-10",
          "title": "Roman J. Israel, Esq.",
          "video": false,
          "vote_average": 0,
          "vote_count": 2,
        },
        Object {
          "adult": false,
          "backdrop_path": "/mJHSPIvKoa1gwrdAMXWWq5utx3b.jpg",
          "genre_ids": Array [
            18,
            53,
          ],
          "id": 345923,
          "original_language": "en",
          "original_title": "All I See Is You",
          "overview": "A blind woman's relationship with her husband changes when she regains her sight and discovers disturbing details about themselves.",
          "popularity": 76.652255,
          "poster_path": "/wc9MgFWuBl3VS72w6VtiZXTgN79.jpg",
          "release_date": "2017-08-10",
          "title": "All I See Is You",
          "video": false,
          "vote_average": 3.4,
          "vote_count": 21,
        },
      ]
    }
  />
</styled.section>
`;
